src/dapp/libexec/dapp/dapp---parse-deps

```diff
@@ -8,8 +8,8 @@ for x; do
   if [[ ${x:?} =~ ^[^/]+/([^/]+)$ ]]; then
     echo "$x" "${BASH_REMATCH[1]}" "https://github.com/$x"
   # matches strings like foo@bar, foo:bar or /foo/bar
-  elif [[ $x =~ .*[@:/](.*) ]]; then
-    name=${BASH_REMATCH[1]%.*}
+  elif [[ $x =~ .*[:/](.*) ]]; then
```

Comment:
Wasn't clear to me why the removed things are needed and were blocking me from getting things working. Pls advise if we need an alternative way to do this or if this looks good

Reply:
@d-xo I'm curious about this as well. What was the purpose originally?

Reply:
Honestly I have no idea

Reply:
lolol oke hope nothing breaks

Reply:
same hehe, but since `parse-deps` is only used in `dapp install` (and `dapp clone` which we can probs just delete), and `dapp install <dep>@<ref>` just always errored out in old versions I figured we were probably OK here.

Reply:
Yeah same thought why it was ok